Problem w/mysql-connector-java-6.0.5.jar waarp 3.0.7

It seems not to be compatible with old mysql setup:
``` 
<db>
        <dbdriver>mysql</dbdriver>
        <dbserver>jdbc:mysql://<ip>:3306/waarp</dbserver>
        <dbuser>waarp</dbuser>
        <dbpasswd>****</dbpasswd>
</db>
```
```
Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'. The driver is automatically registered via the SPI and manual loading of the driver class is generally unnecessary.
29/12/2016/17:34:33.748 ERROR [org.waarp.openr66.server.R66Server] [main] org.waarp.openr66.server.R66Server.main(R66Server.java:63) : Startup of server is in error
java.lang.NoClassDefFoundError: com/mysql/jdbc/jdbc2/optional/MysqlConnectionPoolDataSource
        at org.waarp.common.database.model.DbModelMysql.<init>(DbModelMysql.java:99)
        at org.waarp.openr66.database.model.DbModelMysql.<init>(DbModelMysql.java:58)
        at org.waarp.openr66.database.model.DbModelFactory.initialize(DbModelFactory.java:59)
        at org.waarp.openr66.configuration.FileBasedConfiguration.loadDatabase(FileBasedConfiguration.java:1822)
        at org.waarp.openr66.configuration.FileBasedConfiguration.setConfigurationServerFromXml(FileBasedConfiguration.java:2337)
        at org.waarp.openr66.server.R66Server.initialize(R66Server.java:72)
        at org.waarp.openr66.server.R66Server.main(R66Server.java:54)
Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.jdbc2.optional.MysqlConnectionPoolDataSource
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 7 common frames omitted
```
Any insight of which dbdriver should I use instead?

Regards

Alex

--- a/pom.xml
+++ b/pom.xml
@@ -78,7 +78,7 @@
     <dependency>
     	<groupId>mysql</groupId>
     	<artifactId>mysql-connector-java</artifactId>
-    	<version>6.0.5</version>
+    	<version>[5.1.0,6.0.0)</version>
     	<optional>true</optional>
     </dependency>
     <dependency>

